### DISCUSSION TITLE
R08 - addition to the vocabulary description

### DISCUSSION THREAD

**Discussion by vturpin** [OneArgo/ArgoVocabs#137] [General]
R08 is based on WMO "**Common Code Tables C-3 (CCT C-3): Instrument make and type for water temperature
profile measurement with fall rate equation coefficients**" ; The official code table is available here: https://library.wmo.int/doc_num.php?explnum_id=11283

These tables up to date are pretty hard to find online.
I would like here to keep track on this information and suggest to add it somewhere in the R08 description.

Also I think it is usefull to have an idea of the procedure to add a concept in the CCT C-3:
To suggest an addition to the CCT C-3, follow this exemple: https://github.com/wmo-im/CCT/issues/110 to submit an issue.
When an addition is accepted, the issue on github is closed. 
Then the PR is made within a couple of month to update this table: https://github.com/wmo-im/CCT/blob/master/C03.csv
Then it can take a year to update the document; https://library.wmo.int/doc_num.php?explnum_id=11283

Note that R08 can be updated while we are still waiting for WMO CCT C-3 validation.

**Comment by vpaba** [comment]
Hi @vturpin, thanks for this. I'm working on updating the description for R08, and add a link to the original WMO table as you suggest. 

As I found three lists of the same WMO table, I just wanted to confirm that the one you linked above is the one we want to include in the description:

- https://library.wmo.int/doc_num.php?explnum_id=11283 (PDF file)
- https://github.com/wmo-im/CCT/blob/master/C03.csv (CSV file)
- https://www.nodc.noaa.gov/gtspp/document/codetbls/wmocodes/table1770.html (HTML - this is called Table 1770 so not sure if it differs from the CCT C-3 table?)

I'm asking because I assume there is no controlled vocabulary for the whole WMO CCT C-3 table, and the PDF may be the least interoperable amongst those three. But if that's where the original list is kept then happy to include the link to it.

**Comment by delphinedobler** [comment]
R08 definition has been updated, the discussion can be closed.
